### ISSUE TITLE
Ransomware & Windows Defender

### ISSUE THREAD

**Issue by regis586** [lenmus/lenmus#123]
Hi,
Today, I've got the same thing than the closed issue -> https://github.com/lenmus/lenmus/issues/111

![20200706-Defender-Alerte Ransomware-Lenmus-2](https://user-images.githubusercontent.com/67907497/86594451-55e10a80-bf97-11ea-8a23-8894fea22676.jpg)

**Comment by cecilios** [comment]
Thanks for reporting but it is a false positive. Anyway, try checking the binaries with an online antivirus service that uses several antivirus, such as Virus Total (https://www.virustotal.com) or the Jotti malware scanner (https: //virusscan.jotti. org) that quickly check a file against more than 50 virus and malware detectors.

This false positive seems to be caused because both the installer and the uninstaller are created with NSIS software. It is due to the way NSIS works, that Microsoft does't like: http://stackoverflow.com/questions/48833995/ddg#48835095

It seems to be a frequent problem when using NSIS, but it hasn't happened to LenMus until now:
https://nsis.sourceforge.io/NSIS_False_Positives

I have reported the problem to Microsoft and they just answered this:
![image](https://user-images.githubusercontent.com/5238679/86598188-0ef61380-bf9d-11ea-89cd-33a29a461ca5.png)

But I cannot send them more information as I do not use Windows.

This issue of the false positive is a problem for LenMus but I cannot fight against Microsoft. A new Microsoft strategy against free software?

**Comment by regis586** [comment]
Thanks for your return,

It would be highly unlikely for MS, to have such thoughts…

😊

Sent from Mail for Windows 10

**Comment by aubreyz** [comment]
Since I think the software works perfectly as a portable application, it is probably best to get rid of the installer altogether, and just zip it up (or have a simple self extracting exe).

**Comment by cecilios** [comment]
Yes, it can be used as a portable application as installation could be reduced to copying a tree of folders. Apart from the .exe file it is necessary to install some folders with resources (icons, eBooks, translations, ...).

But apart from these files, users expect to find the program in the start menu or, alternatively, to have a shortcut on the screen.

I stopped using Windows more than ten years ago. Current installer dates from then. But I don't know how to do what you suggest. Can a self-extracting zip file install a tree of folders with its content? In that case this could be a solution for installing the program. But, how to add the program to the start menu or, alternatively, create a shortcut on the screen?

**Comment by aubreyz** [comment]
Yes a zip can unpack to a folder structure.  I have no experience of windows installers, but it may be worth trying others. It is possible that the one you chose does have some suspicious behavior that could potentially deliver malware.
